### DISCUSSION TITLE
LibreTiny esphome for HA, please add to Proxmox scripts

### DISCUSSION THREAD

**Discussion by Stofakiller2** [tteck/Proxmox#1847] [Ideas]
Hi,

LibreTiny esphome is widely used for those that uses eg Tuya chipsets, it works like a charm in HA, today as an addon...

Please add it to your great script collection...

https://github.com/libretiny-eu/libretiny-esphome

/Jan

**Comment by tteck** [comment]
[ESPHome 2023.9.0](https://github.com/esphome/esphome/releases/tag/2023.9.0) has this now, correct?

>Support for LibreTiny platform (RTL8710, BK7231 & other modules) [esphome#3509](https://github.com/esphome/esphome/pull/3509) by [@kuba2k2](https://github.com/kuba2k2) (new-integration)

**Comment by Stofakiller2** [comment]
Hi,

Yes correct... Current version: 2023.9.0-dev

Den ons. 27. sep. 2023 kl. 10.07 skrev tteckster ***@***.***>:

**Reply by tteck** [reply]
It's not dev now, https://github.com/esphome/esphome/releases/tag/2023.9.0

**Comment by Stofakiller2** [comment]
No but it’s called that when installed I HA 😊 Just so you don’t get confused when building it 😊

**Reply by tteck** [reply]
I don't believe there is any confusion here. ESPHome version 2023.9.0 already includes LibreTiny support, so the only necessary action is to upgrade ESPHome.

**Comment by Stofakiller2** [comment]
Ahh, i didn’t know that, thanks for the info I’ll try it out later today 👍
